devmgmtui/src/components/filemgmt/FileUploadComponent.js

```diff
@@ -17,12 +17,16 @@ class FileUploadComponent extends Component {
     }
   }
 
-  handleFileInputChange(fileList)  {
-    let files = this.props.filemgmt.uploadableFiles;
-    for (let i = 0; i < fileList.length; i++) {
-      files.push(fileList[i]);
+  handleFileInputChange(fileList, type)  {
+    if(type == "folderinput" && fileList.length <= 0) {
```

Comment:
Why do you really need type? Doesn't fileList.length work? What will be the fileList.length value for an empty folder ?

Reply:
There are two buttons for - one for file upload and the other for folder upload. This button handles events from both the buttons. I wanted to display that alert only in the case of folder upload so I added that extra parameter to differentiate.